content/blog/2015/2015-10-25-jenkins-2-0-proposal-introduce-a-policy-for-api-deprecation.adoc

```diff
@@ -0,0 +1,83 @@
+---
+:layout: post
+:title: "Jenkins 2.0 Proposal: Introduce a policy for API deprecation"
+:nodeid: 636
+:created: 1445833065
+:tags:
+  - general
+  - core
+  - feedback
+:author: rtyler
+---
+Over the past few weeks there has been a vibrant discussion happening on the
+https://groups.google.com/group/jenkinsci-dev/topics[jenkinsci-dev@] mailing
+list as to what "https://wiki.jenkins.io/display/JENKINS/Jenkins+2.0[Jenkins
```

Comment:
I found at least this didn't convert right, but since its old and pointed to the wiki anyways, i'm kinda meh about it.

Reply:
Okay, apparently its because its in quotes, link:https://wiki.jenkins.io/display/JENKINS/Jenkins+2.0[Jenkins would fix it, but hard to be done automatically.